src/__tests__/react-native-api.test.tsx

```diff
@@ -91,3 +81,27 @@ test('React Native API assumption: <TextInput> renders single host element', ()
     />
   `);
 });
+
+test('React Native API assumption: <Switch> renders single host element', () => {
+  const view = render(
+    <Switch testID="test" value={true} onChange={jest.fn()} />
+  );
+  expect(view.getByTestId('test')).toBe(view.getByTestId('test'));
```

Comment:
why did you remove this expectation from the other tests for view and text but not for switch? (i don't really understand the value of this assertion)